pubspec.yaml

```diff
@@ -10,21 +10,19 @@ topics:
   - sms-retriever-api
   - smart-lock
 screenshots:
-  - description: 'SmartAuth Cover'
+  - description: "SmartAuth Cover"
     path: example/smart_auth_cover.png
 funding:
   - https://ko-fi.com/flutterman
 
-
 environment:
-  sdk: ">=2.16.1 <4.0.0"
+  sdk: ">=2.17.0 <4.0.0"
```

Comment:
Is there any problem with `2.16.1`

Reply:
Enhanced enums are part of flutter version 2.17.0 and above.